test/verify/machineslib.py

```diff
@@ -555,7 +555,7 @@ def testAddDisk(self):
         b.wait_in_text("#vm-subVmTest1-disks-vde-target", "vde")
         b.wait_in_text("#vm-subVmTest1-disks-vde-bus", "virtio")
         b.wait_in_text("#vm-subVmTest1-disks-vde-device", "disk")
-        b.wait_in_text("#vm-subVmTest1-disks-vde-source", "/mnt/vm_one/mydiskofpoolone_temporary") # should be gone after shut down
+        b.wait_in_text("#vm-subVmTest1-disks-vde-source", "mydiskofpoolone_temporary") # should be gone after shut down
```

Comment:
This is because it now shows volume names, not paths any more, right?

Reply:
Correct. By default newly created disks will use the pool,volume syntax. One will be able to find out the disk path in the future by introspecting the storage volume the disk is on.